### ISSUE TITLE
Don't start training from a random protocol

### ISSUE THREAD

**Issue by v-sivak** [v-sivak/gkp-rl#8]
Two ways to approach this:
1) do a supervised pre-training of the policy network to reproduce the actions of a good protocol. Good protocol can be obtained using GRAPE or RL with 'overlap' reward.
2) Instead of pre-training the neural net, can simply add the NN output to the scripted protocol in the action wrapper. This option seems easier.
